src/feature_flags.ts

```diff
@@ -33,6 +33,8 @@ export const defaultFlags = {
 
   // Create unique entry file instead of a file that is based on the function name.
   zisi_unique_entry_file: false,
+
+  zisi_esbuild_upstream: false,
```

Comment:
we can leverage real feature flags from devcycle as well no need for defining constants here